charts/astronomer/templates/_helpers.yaml

```diff
@@ -167,6 +167,32 @@
 {{- end }}
 {{- end }}
 
+{{ define "commander_metadataEnv" -}}
+- name: COMMANDER_AIRFLOW_CHART_VERSION
+  value: {{ .Values.airflowChartVersion | default "" | quote }}
+- name: COMMANDER_DATAPLANE_CHART_VERSION
+  value: {{ .Values.commander.dataplaneChartVersion | default "" | quote }}
```

Comment:
@danielhoherd @Shubham-astro dataplaneChartVersion should be similar numbering as control plane helm chart version right like {{ .Chart.Version }}